tools/documentation/pkgdown.yml

```diff
@@ -161,6 +161,7 @@ reference:
       - createWebDependency
       - resourcePaths
       - registerInputHandler
+      - registerThemeDependency
```

Comment:
Is this really needed? I don't think pkgdown requires `@keywords internal` functions here (but maybe our custom pkgdown site does?)

Reply:
Hm, that might have been from before it had `@keywords internal`. I'll remove it.